### ISSUE TITLE
脱ブラックボックス　5.4 ネットワークの訓練

### ISSUE THREAD

**Issue by NorikatsuMiyoshi** [kikagaku/kikagaku-public-issues#53]
大変楽しく勉強させていただいております。ありがとうございます。5.4ネットワークの訓練のところで、正規化に関してなのですが、batch normalizationを用いるところがよくわかりません。何か参考にすべきものがございましたらご教授いただけないでしょうか？（BatchNorm1dのエラーに対する対策に苦慮しております。）大変申し訳ございませんが、何卒よろしくお願い申し上げます。

**Comment by kikagaku** [comment]
ご質問いただき、ありがとうございます。
下記のリンクなど参考に勉強していただけると理解が深まるかと思いますので試行錯誤してみてください。

今後ともよろしくお願いいたします。
https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html

**Comment by NorikatsuMiyoshi** [comment]
キカガクの皆様
どうもありがとうございます。勉強させていただきます。三吉範克


2021年4月21日(水) 20:55 kikagaku ***@***.***>:
